aws-ssm-parameter/src/main/java/com/amazonaws/ssm/parameter/BaseHandlerStd.java

```diff
@@ -88,6 +94,11 @@ protected static boolean stabilize(
         if(response == null || response.invalidParameters().size() != 0) {
             return false;
         }
+
+        // Temporary logging, TODO: Remove these logs after debugging
```

Comment:
How long do you plan to keep this logging?

Reply:
Probably till the migration is completed, once it is done, we plan on removing this

Reply:
Why are we logging GetParameter response?

Reply:
Updated to only log name and version from get parameters response